modules/nf-core/fast2q/main.nf

```diff
@@ -0,0 +1,38 @@
+process FAST2Q {
+
+    tag "2FAST2Q"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/fast2q' :
+        'biocontainers/2.7.2--pyh7e72e81_0' }"
+
+    input:
+    tuple val(meta1), path(fastq)
+    tuple val(meta2), path(library)
+
+    output:
+    tuple val(meta), path("./2FAST2Q_output/*.*")   , emit: processed
+    path "versions.yml"                             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args            = task.ext.args ?: ''
+    def input_file      = "--s ${fastq}" ?: ''
+    def library_file    = (library instanceof Path && library.exists()) ? "--g ${library}" : ''
+
+    """
+    mkdir -p ./2FAST2Q_output
```

Comment:
Is there a reason to make a subdirectory, as opposed to just making the files inside the working directory?
What files does it output? 
Could it use `prefix` to name them?